Languages/IronPython/Tests/interop/net/derivation/test_property_override.py

```diff
@@ -78,7 +77,10 @@ def get_StrProperty(self):
     x = C()
     p = IProperty11.StrProperty
     p.__set__(x, 'abc')     # no-op equivalent?
-    AssertError(SystemError, lambda: p.SetValue(x, 'def'))  # ?
+    if is_netstandard: # no System.SystemException in netstandard1.6
```

Comment:
There is a `SystemError` in the .NET Core build, it's just associated with a `SystemException` that we defined ourselves (in IronPython/Runtime/Exceptions/SystemException.cs) instead of the one which is a base class of `MethodAccessException`.

We might be able to get around it in some places but I wouldn't jump through hoops to get it working. They're adding `System.SystemException` back in netstandard2.0 so the whole problem will go away.